features/templates.feature

```diff
@@ -12,3 +47,36 @@ Feature: Templates
           | template | number |
           |  valid   |   no   |
           |  invalid |  some  |
+
+    @ignore
+    Scenario Outline: Merges template into user's config
+        Given an existing pipeline with job main
+        And an existing template called Test
+        And user specifies template Test in job main
+        When user starts the pipeline
+        Then job main executes what is specified in template
+
+      @ignore
+      Scenario Outline: Job config takes precedence over template config
+          Given an existing pipeline with job main
+          And an existing template called Test
+          And user specifies template Test in job main
+          And user has some settings defined
+          And template Test has the same settings with different values
+          When user starts the pipeline
+          Then settings is the job settings
+
+    @ignore
+    Scenario Outline: Publish template saves template into Datastore
+        Given a template with a(n) <format> format
+        And it has the <permission> permissions to publish
+        And the template does not exist
```

Comment:
> And the template does not exist

Is this referring to a completely brand new template, or a template+version does not exist?